src/Generator/Generators/CSharp/CSharpCommentPrinter.cs

```diff
@@ -25,15 +25,49 @@ private static void GetCommentSections(this Comment comment, List<Section> secti
                 case DocumentationCommentKind.FullComment:
                     var fullComment = (FullComment) comment;
                     foreach (var block in fullComment.Blocks)
+                    {
                         block.GetCommentSections(sections);
+                        if (!string.IsNullOrEmpty(sections.Last().CurrentLine.ToString()))
+                            sections.Last().NewLine();
+                    }
                     break;
                 case DocumentationCommentKind.BlockCommandComment:
                     var blockCommandComment = (BlockCommandComment) comment;
-                    if (blockCommandComment.CommandKind == CommentCommandKind.Return &&
-                        blockCommandComment.ParagraphComment != null)
+                    if (blockCommandComment.ParagraphComment != null)
                     {
-                        sections.Add(new Section(CommentElement.Returns));
-                        blockCommandComment.ParagraphComment.GetCommentSections(sections);
+                        switch (blockCommandComment.CommandKind)
+                        {
+                            case CommentCommandKind.Brief:
+                                if (sections.Count == 1)
```

Comment:
This whole block can be simplified because comment sections are obtained in both cases.